addons/account/views/audit_trail_report_views.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_message_tree_audit_log" model="ir.ui.view">
+        <field name="name">mail.message.tree.inherit.audit.log</field>
+        <field name="model">mail.message</field>
+        <field name="priority">99</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom" delete="1" create="0" action="action_open_document" type="object">
+                <field name="res_id" column_invisible="True"/>
+                <field name="date" readonly="1"/>
+                <field name="author_id" widget="many2one_avatar" readonly="1"/>
+                <field name="account_audit_log_move_id"/>
+                <field name="account_audit_log_preview"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_message_tree_audit_log_search" model="ir.ui.view">
+        <field name="name">mail.message.search</field>
+        <field name="model">mail.message</field>
+        <field name="priority">99</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <search string="Messages Search">
+                <field name="account_audit_log_move_id"/>
+                <field name="author_id"/>
+                <field name="date" string="Date"/>
+                <filter string="Updated Entries"
+                        name="audit_updated_entries"
+                        domain="[('tracking_value_ids', '!=', False)]"
+                        groups="base.group_system"/>
+                <filter string="Cancelled"
```

Comment:
Well ... looks like a really bad idea to me. This will be both not performant both error prone.

Reply:
@tde-banana-odoo Which one are you talking about? the domain/groups? And why?

Reply:
It's the only field that contains the tracking value information we need though. And since it's just a filter which is rarely executed, it's fine to have slow calculation here isn't it?